src/main/java/com/aspire/student/controller/StudentController.java

```diff
@@ -148,7 +148,8 @@ public void saveStudent(ActionRequest actionRequest, ActionResponse actionRespon
     try {
       student = studentService.saveStudent(student);
     } catch (Exception e) {
-      logger.error("Error while saving student", e);
+      //logger.error("Error while saving student", e);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') This block of commented-out lines of code should be removed. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/coding_rules#rule_key=squid%3ACommentedOutCodeLine)